pkgs/by-name/mi/minijinja/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "minijinja";
+  version = "1.0.12";
+
+  src = fetchFromGitHub {
+    owner = "mitsuhiko";
+    repo = pname;
```

Please keep the repo name static
```suggestion
    repo = "minijinja";
```

---

Thanks, fixed.